### DISCUSSION TITLE
How to style the output PDF

### DISCUSSION THREAD

**Discussion by commandcenterio** [remarkjs/remark#1297] [Q&A]
I am taking markdown and converting to pdf and docx, then downloading the blob in the client.
I was hoping to style the pdf/docx but was unable to find any documentation on that subject. 
Is this possible?

**Comment by ChristianMurphy** [comment]
Welcome @commandcenterio! 👋 

> I am taking markdown and converting to pdf and docx, then downloading the blob in the client.

Remark does not do this, could you clarify on what tools you are using?

> I was hoping to style the pdf/docx but was unable to find any documentation on that subject.

Remark doesn't do this. We don't document features which aren't part of the project 😉 

> Is this possible?

Sure
The approach I'd recommend would be to render the Markdown to HTML.
Style the document how you want it to appear with CSS. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Printing
Then print it with a browser, this could be automated with a tool like Playwright https://playwright.dev/docs/api/class-page#page-pdf
